cms/static/js/views/license-view.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * Provide license display view for license models.
+ * The license display view is used to preview the selected license in
+ * forms for new or existing courses and videos. Additionally, the display
+ * view is used to preview the license underneath videos and inside the
+ * course outline. Such that the author knows under which license the
+ * course material will be published.
+ */
+define(["js/views/baseview", "underscore", "gettext", "js/models/license"],
+    function(BaseView, _, gettext, LicenseModel) {
+
+        var LicenseView = BaseView.extend({
+            initialize: function(options) {
+                if (!this.model) {
+                    this.model = new LicenseModel();
+                }
+                else if (!(this.model instanceof LicenseModel)) {
```

Comment:
See earlier comment about the view altering the model that was passed in. Very, very bad idea. Don't do this.